grc/utils/form_custom_validators.py

```diff
@@ -265,7 +265,9 @@ def fileSizeLimit(max_size_in_mb):
 
     def file_length_check(form, field):
         for data in field.data:
-            if len(data.read()) > max_bytes:
+            if len(data.read()) == 0:
+                raise ValidationError(f'The selected file is zero bytes')
+            elif len(data.read()) > max_bytes:
```

Comment:
A couple of points:
* Is it safe to do `data.read()` twice in a row?
(I assume it is, but just wanted to check we don't need to seek back to the start)
* The error message couple be clearer - I'd check with Spence or Alex when they're back, but maybe something like `The selected file is empty. Check that the file you are uploading has the content you expect`

Reply:
Actually, turns out I did need a `data.seek(0)`...